site/app/controllers/admin/LateController.php

```diff
@@ -226,6 +226,13 @@ public function updateExtension() {
                     JsonResponse::getFailResponse($error)
                 );
             }
+            $reason_for_extension = null;
+            if (isset($_POST['reason_for_exception'])) {
+                $reason_for_exception = $_POST['reason_for_exception'];
+            }
+            else {
+                $reason_for_exception = 'unspecified';
+            }
```

Comment:
This is a great chance to use PHP's [null coalescing](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) operator!

```suggestion
            $reason_for_exception = $_POST['reason_for_exception'] ?? 'unspecified';
```

I assume the "exception" vs "extension" naming thing here is a typo?